trieste/models/gpflow/models.py

```diff
@@ -277,8 +286,17 @@ def trajectory_sampler(self) -> TrajectorySampler[GaussianProcessRegression]:
 
         :return: The trajectory sampler.
         """
+
+        return RandomFourierFeatureTrajectorySampler(self, self._num_rff_features)
+
+    def get_internal_data(self) -> Dataset:
+        """
+        Return the model's training data.
+
+        :return: The model's training data.
+        """
         models_data = Dataset(self.model.data[0].value(), self.model.data[1].value())
```

Comment:
(note sure there's anything gained from setting to a variable)

```suggestion
        return Dataset(self.model.data[0].value(), self.model.data[1].value())
```